src/components/Leaderboard/Leaderboard.module.css

```diff
@@ -0,0 +1,44 @@
+.leaderboardPage {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+    
+.leaderboard {
+    height: 60vh;
+    display: flex;
+    flex-direction: column;
+    padding: 10vmin 20vmin;
+    background-color: white;
+    font-size: 2em;
+    overflow-y: scroll;
+    overflow-y: auto;
+    list-style: none;
+    border: 5px solid black;
+    border-radius: 20px;
+}
+    
+table {
```

Comment:
Does this and the ones below affect the whole project?

Reply:
If so they can probably live in the index.css so easier to find maybe?

Reply:
idk, but I should probably change them to unique names just in case

Reply:
Yeah classes would probably be better